app_base.htm

```diff
@@ -0,0 +1,79 @@
+<%inherit file="/homepage/templates/base.htm"/>
+
+<%block name="myheader">
+  <nav class="navbar navbar-inverse">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <i class="fa fa-headphones fa-4x" style="color:#3D5A67;"></i>
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
```

Comment:
What's with the repeated icon bars?

Reply:
when the screen goes small it turns into the three bar things to click on to expand